tests/fv/__init__.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sh
+from sh import docker
+
+
+def setup_package():
+    pwd = sh.pwd().stdout.rstrip()
+
+    docker_exec = docker.bake("exec")
+    host1_exec = docker_exec.bake("-t", "host1", "bash", "-c")
```

Comment:
Not sure that docker jenkins has anything to do with it since anyone can run the FVs.  I'm ok with the interpolation being broad though - although we probably should indicate that running the FVs will tear down all existing containers (one for the list though I suspect)